### PR TITLE
Elin Elmvik Diczfalusy & Aleksandra Safranko Chatbot Project/Week 4

### PR DESCRIPTION
**Pull request by A1eksa**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,25 +1,159 @@
...
+  currentQuestionNumber++
+}
+
+const handleNameInput = () => {
+  const name = nameInput.value;
+  showMessage(name, 'user')
+  nameInput.value = '';
+  setTimeout(() => showFlowerOptions(name), 500)
+
+  // Function that changes layout to three buttons
```
Comment (hemmahosjessi) [inline code/script.js:50]: Really good with the comments to easily follow what does what... ⭐️

File: code/script.js [hunk 2]
```diff
@@ -1,25 +1,159 @@
...
+      console.log("This is", romanticBth.name)
+      showMessage(chosenEvent, 'user')
+      currentQuestion()
+      setTimeout(() => showMessage(`So you chosen ${chosenEvent} event. What size do you need on the bouquet?`, 'bot'), 500)
+    })
+
+  }
+
+};
+
```
Comment (Mattsson717) [inline code/script.js:92]: You could skip the spaces between the bracets just to make it look more clean. :)

File: code/style.css [hunk 3]
```diff
@@ -138,3 +167,20 @@ button:hover {
...
+  width: 20vh;
+  
+}
+
+footer {
+  display: block;
+  color: black;
+  border:none;
+  padding-left: 20px;
+}
```
Comment (hemmahosjessi) [inline code/style.css]: Not sure if it is important, but it says it needs a blank space...

File: README.md [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Practice some parts of JavaScript by creating a chat bot, that will respond on three questions. The project is created with pair-programming
 
 ## The problem
+We got 5 iteration that we needed to go through and we also got some code to begin with.
+
+So with javaScript we changed the behaivour of the page. The work flow was to listen to an event, and then trigger some answer depending on answer från the user and then ask different questions.
+
+ he hardest part was to create crete new buttons within the javaScript with innerHTML and not make the code to long.
```
Comment (Mattsson717) [inline README.md:10]: The buttons looked great :)

**Review by Mattsson717 — COMMENTED** [review]
Great work! You completed the requirements and the code was easy to follow with your comments.

**Review by hemmahosjessi — COMMENTED** [review]
GREAT WORK! Not much to comment on. ⭐️ 
When reading your "hardest part" it seems you managed to solve it perfectly!

**Pull request closed without merging**